apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -143,6 +158,15 @@ export function Configurator({ title }: { title: string }) {
 
         <CurrencyInputControl label="Buy token" tokenIdState={buyTokenState} tokenAmountState={buyTokenAmountState} />
 
+        <Divider variant="middle">Advanced</Divider>
```

Comment:
@fairlighteth could you please help here?
The sidebar goes out of my screen and it's not scrollable. Probably we should make it scrollable or reorganize its structure.